src/core/geometry/qgsgeometry.h

```diff
@@ -2138,7 +2138,7 @@ class CORE_EXPORT QgsGeometry
      *
      * \since QGIS 3.14
```

Comment:
Add "since QGIS 3.32 the optional \a options argument can be used to specify flags which control how the geometry is coerced."